libafl/src/mutators/token_mutations.rs

```diff
@@ -1144,9 +1129,7 @@ where
         // println!("orig: {:#?} new: {:#?}", orig_cmpvals, new_cmpvals);
 
         // Compute when mutating it for the 1st time.
-        if stage_idx == 0 {
-            self.text_type = check_if_text(orig_bytes, orig_bytes.len());
-        }
+        self.text_type = Some(check_if_text(orig_bytes, orig_bytes.len()));
```

Comment:
@tokatoka can you please double check if that's what this is supposed to do?

Reply:
not it's not the same but you can do

if self.text_type == None {
 self.text_type = Some(check_if_text(...))
}

Reply:
Hmm it's different, the idx 0 will run for newly scheduled testcases, while the condition won't know when a new testcase got scheduled. We'll have to look if the current_corpus_idx changed here, right?

Reply:
yes you are right.
what i want to do here is to check it's text type for every new testcase.

Reply:
Done, PTAL